tutorials/plot_point_spread.py

```diff
@@ -0,0 +1,147 @@
+"""
+.. _point_spread:
+
+Corrupt known signal with point spread
+======================================
+
+The aim of this tutorial is to demonstrate how to put a known signal at a
+desired location(s) in a SourceEstimate and then corrupt the signal with
+point-spread by applying a forward and inverse solution.
+"""
+
+import os.path as op
+
+import numpy as np
+
+import mne
+from mne.datasets import sample
+
+from mne.minimum_norm import read_inverse_operator, apply_inverse
+from mne.simulation import simulate_stc, simulate_evoked
+
+###############################################################################
+# Parameters
+
+seed = 8675309
+
+# regularization parameter for inverse method
+method = 'sLORETA'
+snr = 3.
+lambda2 = 1.0 / snr ** 2
+
+# signal simulation parameters
+# do not add extra noise to our know signals
+noise_snr = np.inf
```

Comment:
The name `noise_snr` makes my head hurt by some sort of recursion. How about just `snr`?